app/src/main/java/org/sopt/androidseminar1/SignInActivity.kt

```diff
@@ -0,0 +1,40 @@
+package org.sopt.androidseminar1
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import org.sopt.androidseminar1.databinding.SignInBinding
+
+class SignInActivity: AppCompatActivity() {
+    private lateinit var  binding: SignInBinding // 고정적인 Class 값은 아니다.
+                                                        // activity_main -> ActivityMain
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = SignInBinding.inflate(layoutInflater) // layoutInflater : xml을 불러주는 정도
+        setContentView(binding.root) // setContentView : xml을 그려주는 함수
+
+
+        // intent 변경
+        val goSignup: Intent = Intent(this, SignUpActivity::class.java)
+        val goHomeActivity: Intent = Intent(this, HomeActivity::class.java )
```

Comment:
코틀린에서는 타입 추론 기능을 지원해주기때문에
```suggestion
        val goSignup = Intent(this, SignUpActivity::class.java)
        val goHomeActivity = Intent(this, HomeActivity::class.java )
```
이렇게만 작성하셔도 괜찮습니당~:)